bot/cogs/sync/syncers.py

```diff
@@ -224,4 +232,4 @@ def get_users_for_sync(
             }
         )
 
-    return (len(users_to_create), len(users_to_update))
+    return len(users_to_create), len(users_to_update), None
```

Comment:
Making it an integer like the others in the tuple will just make life easier.

```suggestion
    return len(users_to_create), len(users_to_update), 0
```

Reply:
I figured that would make it more confusing instead. Not here, but in the logs. See the other comment.